include/seqan3/alphabet/structure/dot_bracket3.hpp

```diff
@@ -113,23 +102,23 @@ class dot_bracket3 : public alphabet_base<dot_bracket3, 3>
      */
     constexpr bool is_pair_open() const noexcept
     {
-        return *this == PAIR_OPEN;
+        return this->to_char() == '(';
```

Comment:
hm I wonder if the extra call to `to_char` makes a performance difference compared to directly comparing the rank with `this->rank == 1` :thinking:

Reply:
Good point, so I will just compare the rank value.